test/test_kaldi_dirs.py

```diff
@@ -0,0 +1,305 @@
+import pytest
+import os.path
```

Comment:
People usually use `import os`, I think.
After that, you can use `os.path` in the code.

Also, people usually sort the imports within a block alphabetically, i.e.,
```python3
import contextlib
import os
import pytest
```